### PR TITLE
Week1: Exercise 1.2 

### PR DESCRIPTION
**Pull request by AakashPandya**
Output will look like this: 

image

### CODE REVIEW

File: Week-1/Exercise-1.2/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            <i class="las la-play"><span>Play all</span></i>
+            <i class="las la-plus"><span>Add to</span></i>
+            <i class="las la-ellipsis-h"><span>More</span></i>
+        </div>
+    </header>
+
+    <main class="main-wrapper">
+        <div class="table-container">
+            <div class="table-row">
+                <div class="table-cell">1. One Dance</div>
```
Comment (RajeshSivanesan) [inline Week-1/Exercise-1.2/index.html:37]: @AakashPandya can you use ul..li in place of divs (semantic tags)

**Review by RajeshSivanesan — COMMENTED** [review]
@AakashPandya Added comments, can you please check ?

File: Week-1/Exercise-1.2/index.html (hunk 1 above)
Comment (AakashPandya) [reply Week-1/Exercise-1.2/index.html:37]: At parent level, I've fixed by ul...li
Not sure if you're talking about cells...

**Comment by AakashPandya** [comment]
> @AakashPandya Added comments, can you please check ?

Hi @RajeshSivanesan , Could you please re-review, I have replaced table-row divs with li . Thanks

**Review by RajeshSivanesan — APPROVED** [review]

**Pull request merged**